packages/radix-vue/src/Dialog/Dialog.test.ts

```diff
@@ -129,5 +129,11 @@ describe('given a default Dialog', () => {
         expect(document.body.innerHTML).not.toContain(closeButton.innerHTML)
       })
     })
+
+    it('should focus trigger after closing', async () => {
+      await fireEvent.click(closeButton)
+      await nextTick()
+      expect(document.activeElement?.textContent).toBe(trigger.element.textContent)
```

Comment:
Similar "magic" as [here](https://github.com/radix-vue/radix-vue/pull/519). `document.activeElement` doesn't reflect the correct focused element, but `textContent` contains correct value.